cloud/aws-common/src/main/java/org/apache/druid/common/aws/AWSProxyConfig.java

```diff
@@ -54,4 +54,15 @@ public String getPassword()
   {
     return password;
   }
+
+  @Override
+  public String toString()
+  {
+    return "AWSProxyConfig{" +
+           "host='" + host + '\'' +
+           ", port=" + port +
+           ", username='" + username + '\'' +
+           ", password='" + password + '\'' +
```

Comment:
Minor nit: Should we toString the password here.

Reply:
Good catch, removed it.